lib/LANraragi/Plugin/Metadata/EHentai.pm

```diff
@@ -106,24 +110,30 @@ sub get_tags {
 
 sub lookup_gallery {
 
-    my ( $title, $tags, $thumbhash, $defaultlanguage, $ua, $domain, $usethumbs) = @_;
+    my ( $title, $tags, $thumbhash, $ua, $domain, $defaultlanguage, $jpntitle, $additionaltags, $usethumbs,
```

Comment:
You don't actually need `$jpntitle` and `$additionaltags` in this sub -- It'd be better to not include them since the signature is getting super long already 😛

Reply:
I think I included them to pass them to the function that calls the EH API.

Reply:
Aye, it's needed for `get_tags_from_EH`, which isn't called from `lookup_gallery` but from the base `get_tags` sub.

Reply:
You are right. The function call in line 73 should be changed, too.